openedx_learning/contrib/tagging/readme.rst

```diff
@@ -0,0 +1,26 @@
+Tagging App
+==============
+
+The ``tagging`` app will enable content authors to tag pieces of content and quickly
+filter for ease of re-use.
+
+Motivation
+----------
+
+Tagging content is central to enable content re-use, facilitate the implementation
+of flexible content structures different from the current implementation and
+allow adaptive learning in the Open edX platform.
+
+This service has been implemented as pluggable django app. Since it is necessary for
+it to work independently of the content to which it links to.
+
+Setup
+---------
+
+**TODO:** We need to wait the discussion of the [Taxonomy discovery](https://docs.google.com/document/d/13zfsGDfomSTCp_G-_ncevQHAb4Y4UW0d_6N8R2PdHlA/edit#heading=h.o6fm1hktwp7b)
```

Comment:
nit: [Links in RST](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Reference/Content/Links.html) don't look like Markdown:

```suggestion
**TODO:** We need to wait the discussion of the `Taxonomy discovery <https://docs.google.com/document/d/13zfsGDfomSTCp_G-_ncevQHAb4Y4UW0d_6N8R2PdHlA/edit#heading=h.o6fm1hktwp7b>`_.
```

Reply:
Oh, thanks! My mistake